packages/backend/src/tools/updateDiffHistory.ts

```diff
@@ -27,6 +27,7 @@ async function updateDiffHistoryFile() {
   }
   const chainId = ChainId.fromName(chainName)
 
+  // Get discovered.json from main branch and compare to current
```

Comment:
Since this script is getting more and more useful and the amount of features that it supports is only going up not down I'd extract the code that this comment is referencing and put it in a function called something like `compareDiscoveredWithMaster()`, this way it's easier to see what's going on. The same with the newly added code.